packages/react-ui/src/widgets/CategoryWidgetUI/CategoryWidgetUI.js

```diff
@@ -456,7 +472,8 @@ CategoryWidgetUI.defaultProps = {
   selectedCategories: [],
   animation: true,
   filterable: true,
-  searchable: true
+  searchable: true,
+  locale: 'en-US'
```

Comment:
Maybe it's better to use `DEFAULT_LOCALE` from the `packages/react-ui/src/hooks/useImperativeIntl.js` file

Reply:
yes, you right, I'll change it

Reply:
Done